src/main.ts

```diff
@@ -324,38 +325,45 @@ export default function main() {
 
     userId = resolveUser(userId);
 
+    const existingAuth = getAuthToken(userId);
```

Comment:
This can throw, but we have no try/catch

Reply:
Can't throw anymore.